### DISCUSSION TITLE
Dumb question but is there a place where I can find the controls?

### DISCUSSION THREAD

**Discussion by musicman928** [DigitalLogicSimCommunity/Digital-Logic-Sim-CE#20] [Q&A] [answered]
I'm confused how to edit a circuit, does anyone know how?

**Comment by AOx0** [comment, marked as answer]
Hi, to edit an existing custom chip you have to:
  1. Right click its name in the chip bar
  2. Select the view option.
      - Now you should view the chips as it was when you saved it
  3. Make your modifications.
      - Drag anything you need, add inputs, outputs, etc. 
  4. To save the modifications select the Chip > Update option at the top bar.
      - Or if you want to discard the modifications, click exit at the top left. 
      - When editing  (_viewing_) a chip, the exit options acts as the cancel button rather than exit to the main menu
      - In _normal mode_ the exit button takes you to the home screen

Hope this is of help, and so sorry for the late response!

**Reply by musicman928** [reply]
It's no problem. Thank you!
